src/components/Carousel/Carousel.jsx

```diff
@@ -0,0 +1,44 @@
+import Carousel from 'react-bootstrap/Carousel';
+import styled from 'styled-components';
+import gameImg from '../../assets/imagen1.jpg';
+import gameImg2 from '../../assets/imagen2.jpg';
+import gameImg3 from '../../assets/imagen3.jpg';
+
+function CarouselImg() {
```

Comment:
- [ ] It was a great initiative implementing the carousel, but it ial good to stick to the design given as that is what will happen when you eventually start working with a company. Looking at the given design, the top part is divided into two sections with the right sections giving details about the total number of views. That is not the same with your design
Given design
**![Screen** Shot 2023-09-06 at 5 44 07 AM](https://github.com/JavierAybar/React_Games_Gallery/assets/62849872/33cdf9a7-eaf8-47fc-8d62-5ad862165bc0)
Your design
**![Screen** Shot 2023-09-06 at 5 45 10 AM](https://github.com/JavierAybar/React_Games_Gallery/assets/62849872/0906a86b-6aab-4156-9059-16b53eb37dce)